src/Atmos/Model/linear.jl

```diff
@@ -0,0 +1,71 @@
+abstract type AtmosLinearModel <: BalanceLaw
+end
+
+
+vars_state(lm::AtmosLinearModel, T) = vars_state(lm.atmos,T)
+vars_gradient(lm::AtmosLinearModel, T) = @vars()
+vars_diffusive(lm::AtmosLinearModel, T) = @vars()
+vars_aux(lm::AtmosLinearModel, T) = vars_aux(lm.atmos,T)
+vars_integrals(lm::AtmosLinearModel,T) = @vars()
+
+
+update_aux!(dg::DGModel, lm::AtmosLinearModel, Q::MPIStateArray, auxstate::MPIStateArray, t::Real) = nothing
+integrate_aux!(lm::AtmosLinearModel, integ::Vars, state::Vars, aux::Vars) = nothing
+flux_diffusive!(lm::AtmosLinearModel, flux::Grad, state::Vars, diffusive::Vars, aux::Vars, t::Real) = nothing
+function wavespeed(lm::AtmosLinearModel, nM, state::Vars, aux::Vars, t::Real)
+  return soundspeed_air(Float64(T_0))
+end
+
+boundary_state!(nf, lm::AtmosLinearModel, x...) = nothing
+init_aux!(lm::AtmosLinearModel, aux::Vars, geom::LocalGeometry) = nothing
+init_state!(lm::AtmosLinearModel, state::Vars, aux::Vars, coords, t) = nothing
+
+
+function linear_pressure(moisture, orientation, state::Vars, aux::Vars)
+  ρ = state.ρ
+  invρ = inv(ρ)
+  TS = thermo_state(moisture, orientation, state, aux)
+  # TODO: avoid dividing then multiplying by ρ
+  linearized_air_pressure(ρ, state.ρe * invρ, gravitational_potential(orientation, aux), PhasePartition(TS))
+end
```

Comment:
Is this actually linear?

Reply:
I'm not sure? Maybe we should just remove it for now, since it isn't being used.

Reply:
Also this is never actually used. Is that deliberate?